File: php/MathTest.php
// tests/KalkulatorTest.php
<?php

use PHPUnit\Framework\TestCase;

require_once 'Math.php'; // Sesuaikan path

class MathTest extends TestCase {
    private $math;

    protected function setUp(): void {
        $this->math = new Math();
    }

    public function testTambah() {
        $hasil = $this->math->tambah(2, 3);
        $this->assertEquals(5, $hasil);
    }

    public function testKurang() {
        $hasil = $this->math->kurang(5, 2);
        $this->assertEquals(3, $hasil);
    }
}

File: go/math_test.go
package math

import "testing"

func TestTambah(t *testing.T) {
        hasil := Tambah(2, 3)
        if hasil != 5 {
                t.Errorf("Tambah(2, 3) seharusnya 5, tetapi mendapatkan %d", hasil)
        }
}